tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestTlaConstInliner.scala

```diff
@@ -0,0 +1,67 @@
+package at.forsyte.apalache.tla.pp
+
+import at.forsyte.apalache.tla.lir.oper.TlaArithOper.{minus, plus}
+import at.forsyte.apalache.tla.lir.transformations.impl.IdleTracker
+import at.forsyte.apalache.tla.lir.values.TlaInt
+import at.forsyte.apalache.tla.lir._
+import at.forsyte.apalache.tla.lir.convenience.tla
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{BeforeAndAfterEach, FunSuite, Matchers}
+import at.forsyte.apalache.tla.lir.UntypedPredefs._
+
+@RunWith(classOf[JUnitRunner])
+class TestTlaConstInliner extends FunSuite with BeforeAndAfterEach with Matchers {
+  // some expressions to play with
+  private val primitiveValuedConst = tla.name("PrimitiveValuedConst")
+  private val nonPrimitiveValuedConst = tla.name("NonPrimitiveValuedConst")
+  private val nonConst = tla.name("x")
+  private val primitiveValue = tla.int(42)
+  // module-level constant names
+  private val constants: Set[String] = Set(primitiveValuedConst.name, nonPrimitiveValuedConst.name)
+  // constants mapped to primitive values
+  private val constMap = Map(primitiveValuedConst.name -> primitiveValue.untyped().asInstanceOf[ValEx],
+      nonConst.name -> primitiveValue.untyped().asInstanceOf[ValEx])
+  // the inliner
```

Comment:
I don't think this comment helps illuminate anything as is?

```suggestion
```

But it could maybe be useful as:

```suggestion
  // declare the reference to the inliner for use in each test case, initialized to the default value
```